Documentation/UnitTests.md

```diff
@@ -9,50 +9,51 @@ This is an example of a typical `RxSwift` operator unit test:
 
 ```swift
 func testMap_Range() {
-        // Initializes test scheduler.
-        // Test scheduler implements virtual time that is
-        // detached from local machine clock.
-        // This enables running the simulation as fast as possible
-        // and proving that all events have been handled.
-        let scheduler = TestScheduler(initialClock: 0)
-
-        // Creates a mock hot observable sequence.
-        // The sequence will emit events at designated
-        // times, no matter if there are observers subscribed or not.
-        // (that's what hot means).
-        // This observable sequence will also record all subscriptions
-        // made during its lifetime (`subscriptions` property).
-        let xs = scheduler.createHotObservable([
-            next(150, 1),  // first argument is virtual time, second argument is element value
-            next(210, 0),
-            next(220, 1),
-            next(230, 2),
-            next(240, 4),
+    // Initializes test scheduler.
+    // Test scheduler implements virtual time that is
+    // detached from local machine clock.
+    // This enables running the simulation as fast as possible
+    // and proving that all events have been handled.
+    let scheduler = TestScheduler(initialClock: 0)
+    
+    // Creates a mock hot observable sequence.
+    // The sequence will emit events at designated
+    // times, no matter if there are observers subscribed or not.
+    // (that's what hot means).
+    // This observable sequence will also record all subscriptions
+    // made during its lifetime (`subscriptions` property).
+    let xs = scheduler.createHotObservable(
+        Recorded.events(
+            .next(150, 1),  // first argument is virtual time, second argument is element value
+            .next(210, 0),
+            .next(220, 1),
+            .next(230, 2),
+            .next(240, 4),
             completed(300) // virtual time when completed is sent
```

Comment:
this needs to have the `.` before. The reason it works without is that the `completed` one refers to the global free function and not the case. e.g.: 

```swift
Recorded.events(
            .next(150, 1),  // first argument is virtual time, second argument is element value
            .next(210, 0),
            .next(220, 1),
            .next(230, 2),
            .next(240, 4),
            .completed(300) // virtual time when completed is sent
           )
```

Same for the second example.